extension/source/QuillPage/components/ReviewSecretPhrasePanel.tsx

```diff
@@ -1,11 +1,48 @@
-import { ArrowRight } from 'phosphor-react';
 import * as React from 'react';
 
+import { ArrowRight } from 'phosphor-react';
 import Button from '../../components/Button';
 import Range from '../../helpers/Range';
-import QuickRow from './QuickRow';
 
-type ReviewWordState = 'partial' | 'correct' | 'incorrect';
+const WordInReview: React.FunctionComponent<{
+  index: number;
+  sampleIndex: number;
+  word: string;
+  handleGuess: (index: number, isCorrect: boolean) => void;
+}> = ({ index, sampleIndex, word, handleGuess }) => {
+  const [guess, setGuess] = React.useState('');
+
+  React.useEffect(() => {
+    const isCorrect = guess === word;
+    handleGuess(index, isCorrect);
+    // eslint-disable-next-line
+  }, [guess]);
```

Comment:
Is there a reason all hook deps are not in this?
```suggestion
    // eslint-disable-next-line
  }, [guess, index, word, handleGuess]);
```
If so, would be more explicit in the disable, maybe with a clarifying comment.
`eslint-disable-next-line react-hooks/exhaustive-deps`

Reply:
I'm inclined to agree on both points but can't say for sure.

I think this is mergeable as-is though so considering kautuk is going offline now I've made a separate issue to chase this up https://github.com/jzaki/bls-wallet/issues/116.